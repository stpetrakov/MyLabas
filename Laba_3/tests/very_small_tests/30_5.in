32258 25179 25918 27967 500 24987 15743 31925 15392 26031 21700 6826 19507 14299 5705 28729 13934 3250 24686 11792 17751 12015 5133 11891 13231 20897 11786 30852 12965 19851 