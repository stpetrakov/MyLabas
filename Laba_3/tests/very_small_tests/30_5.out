500 3250 5133 5705 6826 11786 11792 11891 12015 12965 13231 13934 14299 15392 15743 17751 19507 19851 20897 21700 24686 24987 25179 25918 26031 27967 28729 30852 31925 32258 